site-manage/site-manage-participant-helper/src/webapp/content/templates/Confirm.html

```diff
@@ -19,8 +19,10 @@ <h3 rsf:id="addconf.confirming">Confirming Add Participant(s)</h3>
 		<div class="sak-banner-info" rsf:id="info">We Have Infos!</div>
 	</div>
 	<div rsf:id="content:">
-		<h4 class="instruction" rsf:id="msg=addconf.finish"></h4>
-		<span rsf:id="emailnoti"></span>
+		<p class="instruction">
+			<span rsf:id="msg=addconf.finish"></span><br/>
```

Comment:
I'd recommend removing the line break (`<br/>`) because at narrow widths, we'll get uneven wrapping:

![confirmation](https://user-images.githubusercontent.com/12685096/77943277-f7ffa100-728a-11ea-974c-0c5ca60e1606.png)

Reply:
Ok, removed.